CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # SAS Viya Monitoring for Kubernetes
 
+## Unreleased
+
+* **Overall**
+
+* **Metrics**
+  * [UPGRADE] - Kube-prometheus-stack has been upgraded from version 45.28.0 to 48.2.3
+  * [UPGRADE] - Prometheus has been upgraded from version 2.44.0 to 2.45.0
```

Comment:
In my test environment, after fresh install, Prometheus was (still) at 2.44.0.

Reply:
Most likely an artifact of not picking up the component_versions.env file.  I suggest taking care of that first and see whether this goes away.

Reply:
No, for some reason, the Helm chart is not picking up the correct version of the Prometheus in the kube-prometheus-stack [values.yaml](https://github.com/prometheus-community/helm-charts/blob/kube-prometheus-stack-48.3.2/charts/kube-prometheus-stack/values.yaml#L2897) file.  I'm going to manually update it to 2.46.0 for the time being.

Reply:
I am now seeing Prometheus 2.46.0 in my test environment.  Please update the CHANGELOG.md entry to reflect that version (if that is our intent).